build.sbt

```diff
@@ -1,174 +1,21 @@
-import sbtassembly.MergeStrategy
-import sbtrelease.ReleasePlugin._
+import Settings._
+import Testing._
 
-name := "cromwell"
+lazy val core = Project(id="core", base=file("core"))
```

Comment:
Can you switch all of these circa 0.12 `Project.apply()` to the 0.13 macro [`project`](http://www.scala-sbt.org/0.13/docs/ChangeSummary_0.13.0.html#.sbt+format+enhancements)? It should make IntelliJ happier.

```scala
lazy val core = (project in file("core"))
```